crates/rome_js_formatter/src/js/module/literal_export_name.rs

```diff
@@ -11,6 +11,6 @@ impl FormatNodeFields<JsLiteralExportName> for FormatNodeRule<JsLiteralExportNam
     ) -> FormatResult<FormatElement> {
         let JsLiteralExportNameFields { value } = node.as_fields();
 
-        Ok(format_string_literal_token(value?, formatter))
+        FormatLiteralStringToken::from_string(&value?).format(formatter)
```

Comment:
I would use `formatted` here. More idiomatic. 

```suggestion
        formatted![formatter, [FormatLiteralStringToken::from_string(&value?)]]
```